src/lib/process/index.ts

```diff
@@ -3,6 +3,8 @@ import { Constants } from './constants.js'
 import { createProcessTransaction, disableProcessTransaction, getVersion } from './api.js'
 import { utf8ToHex } from './hex.js'
 import { mapRestrictions } from './map.js'
+import type { ChainRestrictions } from '../types/restrictions.js'
```

Comment:
Why do we neeed to prefix with .js if file is .ts? try without extension?

Reply:
It doesn't work without extensions, I think because of this?
https://www.typescriptlang.org/docs/handbook/esm-node.html

`relative import paths need full extensions (e.g we have to write import "./foo.js" instead of import "./foo")`